packages/vite/src/node/plugins/reporter.ts

```diff
@@ -66,6 +66,12 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
     )
   })
 
+  const isCompressibleFile = (fileName: string): boolean =>
+    ['css', 'html', 'js', 'json', 'svg', 'txt', 'xml', 'xhtml'].reduce<boolean>(
+      (isText, ext) => isText || fileName.endsWith(`.${ext}`),
+      false,
+    )
```

Comment:
You can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some here.